terraform/azure_server/files/setup_profile.ps1

```diff
@@ -1,25 +1,5 @@
-if (-not (Get-Command choco -ErrorAction SilentlyContinue)) {
```

Comment:
this is now just a script that sets up the profile, tool installation has been moved directly into the install-choco.ps1 script.